wfe/wfe_test.go

```diff
@@ -190,7 +190,7 @@ func (ra *MockRegistrationAuthority) NewCertificate(ctx context.Context, req cor
 }
 
 func (ra *MockRegistrationAuthority) UpdateRegistration(ctx context.Context, reg core.Registration, updated core.Registration) (core.Registration, error) {
-	if reg.Key != updated.Key {
+	if reg.Key.Key != updated.Key.Key {
```

Comment:
This seems wrong - `reg.Key.Key` is of type `interface{}`, so I'm not sure how equality is defined. Could you either explain why this is right, or use the `publicKeysEqual` method on them?